pumps/mongo.go

```diff
@@ -122,7 +217,7 @@ func (m *MongoPump) connect() {
 			"prefix": mongoPrefix,
 		}).Error("Mongo connection failed:", err)
 
-		time.Sleep(5)
+		time.Sleep(5 * time.Second)
```

Comment:
nice, so previously if mongo failed, it slammed it with numerous connect attempts.

Reply:
It still tries forever which IMP needs to be fixed at some stage, but now it actually waits 5 seconds

Reply:
Wow, this is an error that's far too easy to miss. Will check if `go vet` has anything for it.

Reply:
We have a bunch more of those, FYI:

```
tyk/rpc_analytics_purger.go:37:3: time.Sleep(sleep)
tyk-pump/pumps/elasticsearch.go:137:3: time.Sleep(5)
tyk-pump/pumps/influx.go:72:3: time.Sleep(5)
tyk-pump/pumps/mongo.go:125:3: time.Sleep(5)
tyk-pump/pumps/mongo_aggregate.go:333:3: time.Sleep(5)
tyk-pump/pumps/mongo_selective.go:104:3: time.Sleep(5)
tyk-pump/pumps/statsd.go:63:3: time.Sleep(5)
```

Reply:
good spot. I will do a separate PR to fix that.

Reply:
https://github.com/TykTechnologies/tyk-pump/issues/55